packages/shared/src/components/modals/SubmitArticleModal.tsx

```diff
@@ -225,6 +225,7 @@ export default function SubmitArticleModal({
               name="articleUrl"
               inputId="article_url"
               label="Paste post url"
+              autoComplete="off"
```

Comment:
@sshanzel Great fix, I do think we want to remove it everywhere now.
There are some other places where this happens.

Reply:
Oh, I thought it was just from here as it can be useful on login, etc. - should we loop in the design team for this?

Reply:
Great suggestion! Have it applied now 🚢